### ISSUE TITLE
Citra on macOS seems to corrupt games it loads, and then crash when loading them afterwards

### ISSUE THREAD

**Issue by nadiaholmquist** [citra-emu/citra#5799]
When loading a game that has been dumped from the SD card from my 3DS, Citra plays it fine on first load, but on any subsequent attempt to load the game, Citra instead crashes. Trying to load the same game in Citra on Linux after the fact also fails, but a fresh copy of the game files work every time on the Linux version.

![billede](https://user-images.githubusercontent.com/893884/121813587-e9718f80-cc6c-11eb-8a35-026b5cbcb4d1.png)
(that timestamp looks rather old for a nightly, but it is the latest the maintanance tool gave me from updating today)

Output in macOS crash dialog: [citra-crash.txt](https://github.com/citra-emu/citra/files/6644167/citra-crash.txt)
Command line output:
```
dynarmic: macOS MachHandler: Exception was not in registered code blocks (rip 0x00000000000000)
zsh: segmentation fault  ./citra-qt
```

Any other info I can provide?

**Comment by SachinVin** [comment]
Dupe of #5726, Use canary or disable `Disk Shader Cache`.

**Comment by SachinVin** [comment]
Fixed with #5785
